content/applications/finance/accounting/fiscal_localizations/localizations/india.rst

```diff
@@ -0,0 +1,188 @@
+   * - :guilabel:`Indian - Accounting`
+     - `l10n_in`
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - :guilabel:`Indian E-invoice Integration`
+     - `l10n_in_edi`
+     - :ref:`Indian e-invoicing integration <india/e-invoicing>`
+
+.. _india/e-invoicing:
+
+Indian e-invoicing
+==================
+
+Odoo is compliant with the **Indian Good and Services Tax (GST) e-Invoice system** requirements.
+
+.. important::
+   Indian e-invoicing is available from Odoo 15.0. If needed, :doc:`upgrade
+   </administration/upgrade>` your database.
+
+.. _india/e-invoicing-api:
+
+Registration on your NIC e-Invoice web portal
+---------------------------------------------
+
+You must register on the **NIC e-Invoice** web portal to get your **API credentials**. You need
+these credentials to :ref:`configure your Odoo Accounting app <india/e-invoicing-configuration>`.
+
+#. Login to the NIC e-Invoice web portal on - https://einvoice1.gst.gov.in/ by clicking on
+   :guilabel:`Login` and entering your :guilabel:`Username` and :guilabel:`Password`.
+
+   .. note::
+      If you have already registered on the NIC Eway Bill Production portal, then you can use the
+      same login credentials here.
+
+   .. image:: india/e-invoice-system-login.png
+      :align: center
+      :alt: Register Odoo ERP system on e-invoice web portal
+
+#. From your dashboard, go to :menuselection:`API Registration --> User Credentials --> Create API
+   User`.
+
+   .. image:: india/e-invoice-create-api-user.png
+      :align: center
+      :alt: Click on User Credentials and Create API User
+
+#. After that, you receive an :abbr:`OTP (one-time password)` code to your registered mobile number.
+#. Enter the OTP code and click on :guilabel:`Verify OTP`.
+
+   .. image:: india/trigger-otp.png
+      :align: center
+      :alt: Trigger an OTP to your registerd phone number
+
+#. Select :guilabel:`Through GSP` in the first field, select :guilabel:`Tera Software Limited`
+   as your GSP, and type in a :guilabel:`Username` and :guilabel:`Password` for your API.
+
+   .. image:: india/submit-api-registration-details.png
+      :align: center
+      :alt: Submit API specific Username and Password
+
+#. Click on :guilabel:`Submit`.
+
+.. _india/e-invoicing-configuration:
+
+Configuration on Odoo
+---------------------
+
+To set up the e-invoice service, go to :menuselection:`Accounting --> Configuration --> Settings -->
+Indian Electronic Invoicing`, and enter the :guilabel:`Username` and :guilabel:`Password`.
+
+.. image:: india/e-invoice-setup.png
+   :align: center
+   :alt: Setup e-invoice service
+
+.. _india/e-invoicing-journals:
+
+Journals
+~~~~~~~~
+
+To configure your journals, go to :menuselection:`Accounting --> Configuration --> Journals`. Open
+your *sales* journal, and in the :guilabel:`Advanced Settings` tab, under :guilabel:`Electronic Data
+Interchange`, check :guilabel:`E-Invoice (IN)` and :guilabel:`Save`.
```

Comment:
Should not this be automatically the case when you install Indian accounting @nia-odoo @jva-odoo

Reply:
I think so indeed, but only for their main "tax invoice" journal, no? I'll just rephrase it so they can "double-check" and we can always improve this doc later :)